src/main/resources/docs/description/DeclarationOrder.md

```diff
@@ -1,6 +1,6 @@
 Checks the order in which parts of the class or interface declaration are defined.
 
-According to [ Code Conventions for the Java Programming Language][Code Conventions for the Java Programming Language] , the parts of a class or interface declaration should appear in the following order:
+According to  Code Conventions for the Java Programming Language , the parts of a class or interface declaration should appear in the following order:
```

Comment:
Same here, seems the links are getting lost

Reply:
https://github.com/codacy/codacy-checkstyle/pull/36#discussion_r308320334